src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs

```diff
@@ -109,6 +109,12 @@ internal static void RegisterExtensions()
 
 		public static WpfHost? Current => _current;
 
+		/// <summary>
+		/// Gets or sets the current Skia Render surface type.
+		/// </summary>
+		/// <remarks>If <c>null</c>, the host will try to determine the most compatible mode.</remarks>
+		public RenderSurfaceType? RenderSurfaceType { get; set; }
```

Comment:
Should this be made non-nullable with OpenGL being the default value?

Reply:
The idea is to keep consistency with the GTK implementation which uses null for auto detection, as we may add more backends later.